book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ Required ] It would be better if you add a method `add_rental` that can be used any time a book is being rented out, this method will receive as arguments a `person` and a `date` object. Currently, there is no method in your class that can be used to achieve this. You can achieve this by using the intermediate `Rental` class as shown in them method below.

```ruby
  def add_rental(person, date)
     Rental.new(date, self, person)
  end
```
## Project requirements
![Screenshot from 2023-02-11 00-28-03](https://user-images.githubusercontent.com/16392046/218201970-02646983-8ea6-4d04-8028-9e1c91b7e2af.png)